### PR TITLE
Update Timer, Stream, Module API docs

### PR DESCRIPTION
**Pull request by daeyeon**
IoT.js-DCO-1.0-Signed-off-by: Daeyeon Jeong daeyeon.jeong@samsung.com

### CODE REVIEW

File: docs/api/IoT.js-API-Timers.md [hunk 1]
```diff
@@ -11,28 +9,59 @@ The following shows timers module APIs available for each platform.
...
+
+When called, the active Timeout object will not require the IoT.js event loop to remain active. If there is no other activity keeping the event loop running, the process may exit before the Timeout object's callback is invoked.
+
+## Module Functions
 
-#### setTimeout(callback, delay[, args..])
-* `callback: Function([args..])`
-* `delay: Int`
-* Return: `timeoutObject`
+### `setTimeout(callback, delay[, args..])`
```
Comment (chokobole) [inline docs/api/IoT.js-API-Timers.md]: Does `args` belong to `callback`? or do we omit the explanation of `args`?

File: docs/api/IoT.js-API-Timers.md [hunk 2]
```diff
@@ -11,28 +9,59 @@ The following shows timers module APIs available for each platform.
...
+*  `timeoutObject <Timeout>`
 
-Disables timeout call from `setTimeout` method.
+Disables a `Timeout` object from `setTimeout` method.
 
-#### setInterval(callback, delay[, args..])
-* `callback: Function([args..])`
-* `delay: Int`
-* Return: `intervalObject`
+### `setInterval(callback, delay[, args..])`
```
Comment (chokobole) [inline docs/api/IoT.js-API-Timers.md]: ditto

File: docs/api/IoT.js-API-Stream.md [hunk 3]
```diff
@@ -1,107 +1,143 @@
...
+* `size <Number>` - Specify how much data will be read.
+* Returns: `<String> | <Buffer> | <Null>`
 
-#### readable.read([size])
-* `size | Number` - specifies how much data will be read
+The readable.read() method pulls some data out of the internal buffer and returns it. If no data available to be read, null is returned. By default, the data will be returned as a Buffer object.
 
-Reads data from readable stream, and returns the data as `String | Buffer`
+### `readable.resume()`
+* Returns: `this`
```
Comment (chokobole) [inline docs/api/IoT.js-API-Stream.md]: ditto

File: docs/api/IoT.js-API-Stream.md [hunk 4]
```diff
@@ -1,107 +1,143 @@
...
 
+### `'close'`
+* `callback <Function()>`
 
-***
+Emitted when the underlying resource has been closed.
 
-## Class: stream.Writable
+### `'data'`
+* `callback Function(chunk)`
```
Comment (chokobole) [inline docs/api/IoT.js-API-Stream.md]: Please wrap signature with `<>`

**Comment by yichoi** [comment]
LGTM

**Pull request merged**